index.js

```diff
@@ -120,15 +88,13 @@ module.exports = class PriceCheckQuerier {
    * @returns {Promise<string>} a Promise of a string containing the results of the query
    */
   async priceCheckQueryString(query, priorResults, platform = 'pc') {
-    const components = priorResults || await this.priceCheckQuery(query, platform);
+    const components = priorResults || (await this.priceCheckQuery(query, platform));
```

Comment:
Maybe it's not much relevant, but is there any need for the additional parenthesis?

Reply:
I think.this one is a prettier enforcement